### PR TITLE
feat(PlanCard): Adds PlanCard

### PR DESCRIPTION
**Pull request by jcenglish**
### Overview
Adds PlanCard to gamut-labs to be used in Checkout and the Billing page plan change modal.

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/KLXvcxaNkadmU6XFb5ngWO/Pro-Gold-GTM?node-id=695%3A21306&t=gi0TWmyKfBtwq9Si-0
- [x] Related to JIRA ticket: [GRW-424]
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [x] This PR includes testing instructions tests for the code change
- [x] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions
 - PR to use these cards in checkout: https://github.com/codecademy-engineering/mono/pull/1356
 - PR to use these cards in the billing page plan change modal: https://github.com/codecademy-engineering/Codecademy/pull/34733/

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |

[GRW-424]: https://codecademy.atlassian.net/browse/GRW-424?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.285-alpha.8f9ecf.0<br/>@codecademy/gamut-labs@35.5.11-alpha.8f9ecf.0<br/>@codecademy/gamut@52.0.7-alpha.8f9ecf.0<br/>@codecademy/monaco@0.7.7-alpha.8f9ecf.0<br/>@codecademy/styleguide@63.5.9-alpha.8f9ecf.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63f668216c75b60054f4dde4--gamut-preview.netlify.app>https://63f668216c75b60054f4dde4--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63f668216c75b60054f4dde4>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.285-alpha.67f269.0<br/>@codecademy/gamut-labs@35.5.11-alpha.67f269.0<br/>@codecademy/gamut@52.0.7-alpha.67f269.0<br/>@codecademy/monaco@0.7.7-alpha.67f269.0<br/>@codecademy/styleguide@63.5.9-alpha.67f269.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63f67f629f72e306f1a43833--gamut-preview.netlify.app>https://63f67f629f72e306f1a43833--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63f67f629f72e306f1a43833>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.285-alpha.9442d7.0<br/>@codecademy/gamut-labs@35.5.11-alpha.9442d7.0<br/>@codecademy/gamut@52.0.7-alpha.9442d7.0<br/>@codecademy/monaco@0.7.7-alpha.9442d7.0<br/>@codecademy/styleguide@63.5.9-alpha.9442d7.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63f68933b985d00ecf5074d7--gamut-preview.netlify.app>https://63f68933b985d00ecf5074d7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63f68933b985d00ecf5074d7>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.285-alpha.6c1fa5.0<br/>@codecademy/gamut-labs@35.5.11-alpha.6c1fa5.0<br/>@codecademy/gamut@52.0.7-alpha.6c1fa5.0<br/>@codecademy/monaco@0.7.7-alpha.6c1fa5.0<br/>@codecademy/styleguide@63.5.9-alpha.6c1fa5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63f7ba645d4a82021c4000f1--gamut-preview.netlify.app>https://63f7ba645d4a82021c4000f1--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63f7ba645d4a82021c4000f1>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.285-alpha.12af40.0<br/>@codecademy/gamut-labs@35.5.11-alpha.12af40.0<br/>@codecademy/gamut@52.0.7-alpha.12af40.0<br/>@codecademy/monaco@0.7.7-alpha.12af40.0<br/>@codecademy/styleguide@63.5.9-alpha.12af40.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63f7c6b981c30309115003e0--gamut-preview.netlify.app>https://63f7c6b981c30309115003e0--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63f7c6b981c30309115003e0>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.285-alpha.0a95d7.0<br/>@codecademy/gamut-labs@35.5.11-alpha.0a95d7.0<br/>@codecademy/gamut@52.0.7-alpha.0a95d7.0<br/>@codecademy/monaco@0.7.7-alpha.0a95d7.0<br/>@codecademy/styleguide@63.5.9-alpha.0a95d7.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63f8d59fe87a9b27efde9a38--gamut-preview.netlify.app>https://63f8d59fe87a9b27efde9a38--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63f8d59fe87a9b27efde9a38>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.285-alpha.4462b9.0<br/>@codecademy/gamut-labs@35.5.11-alpha.4462b9.0<br/>@codecademy/gamut@52.0.7-alpha.4462b9.0<br/>@codecademy/monaco@0.7.7-alpha.4462b9.0<br/>@codecademy/styleguide@63.5.9-alpha.4462b9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63f943f825c12514aaf0224b--gamut-preview.netlify.app>https://63f943f825c12514aaf0224b--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63f943f825c12514aaf0224b>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.287-alpha.340c99.0<br/>@codecademy/gamut-labs@35.6.2-alpha.340c99.0<br/>@codecademy/gamut@52.0.7-alpha.340c99.0<br/>@codecademy/monaco@0.7.7-alpha.340c99.0<br/>@codecademy/styleguide@63.5.12-alpha.340c99.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63fcd046c6b316284eb6cf51--gamut-preview.netlify.app>https://63fcd046c6b316284eb6cf51--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63fcd046c6b316284eb6cf51>Deploy Logs</a></p>

File: packages/gamut-labs/src/PlanCard/PricingAmount.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+interface PricingAmountProps {
+  product: PlanType;
+  price: string;
+  monthlyPrice?: string;
+  currency: Currency;
+  termMonths?: number;
+  compact?: boolean;
+}
+
+export const PricingAmount: React.FC<PricingAmountProps> = ({
```
Comment (jcenglish) [inline packages/gamut-labs/src/PlanCard/PricingAmount.tsx:28]: PricingAmount is used in the Pricing page too, but I didn't remove that from the monorepo yet. I'll add the PricingAmount component to gamut-labs in a separate PR.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.287-alpha.98740d.0<br/>@codecademy/gamut-labs@35.6.2-alpha.98740d.0<br/>@codecademy/gamut@52.0.7-alpha.98740d.0<br/>@codecademy/monaco@0.7.7-alpha.98740d.0<br/>@codecademy/styleguide@63.5.12-alpha.98740d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63fd379ba4d3420fc6fab5fb--gamut-preview.netlify.app>https://63fd379ba4d3420fc6fab5fb--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63fd379ba4d3420fc6fab5fb>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.288-alpha.3ac977.0<br/>@codecademy/gamut-labs@35.6.3-alpha.3ac977.0<br/>@codecademy/gamut@52.0.7-alpha.3ac977.0<br/>@codecademy/monaco@0.7.7-alpha.3ac977.0<br/>@codecademy/styleguide@63.5.13-alpha.3ac977.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63fe21affd128403fde498b9--gamut-preview.netlify.app>https://63fe21affd128403fde498b9--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63fe21affd128403fde498b9>Deploy Logs</a></p>

**Review by xendke — APPROVED** [review]
this looks good! i think some of the constants like the bulleted features could be props in the future but right now, theyre not changing often or changing between use cases so this is fine with me 👍

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.290-alpha.68bf97.0<br/>@codecademy/gamut-labs@35.6.5-alpha.68bf97.0<br/>@codecademy/gamut@52.0.9-alpha.68bf97.0<br/>@codecademy/monaco@0.7.9-alpha.68bf97.0<br/>@codecademy/styleguide@63.5.15-alpha.68bf97.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://640615d767c91904bc8e0fae--gamut-preview.netlify.app>https://640615d767c91904bc8e0fae--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/640615d767c91904bc8e0fae>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]

**Pull request merged**